Tests/scripts/create_instance.sh

```diff
@@ -20,11 +20,14 @@ then
 fi
 
 MACHINE_STATE=""
-while [ "$MACHINE_STATE" != "fulfilled" ] ; do
-    echo "Waiting for machine to be ready ($REQUEST_ID)."
+TRY_COUNT=1
+MAX_TRIES=10
+while [ "$MACHINE_STATE" != "fulfilled" ] && [[ $TRY_COUNT -le $MAX_TRIES ]]; do
+    echo "Waiting for machine to be ready ($REQUEST_ID). try # $TRY_COUNT"
+    sleep 10
```

Comment:
not super critical - but do you think we should wait first?
if usually the first time succeed - lets wait after.. its only 10 secs but it will happen every build